src/styles/containerComposeStyles/index.ts

```diff
@@ -1,7 +1,6 @@
-import styles from '@styles/styles';
-import ContainerComposeStyles from './types';
+import Styles from '@styles/styles';
```

Comment:
it is not a type.
```suggestion
import styles from '@styles/styles';
```

Reply:
but it's kind a of a module now, no?

Reply:
As I see it is not a module.

https://github.com/Expensify/App/blob/f5194e1d349b0f7bb80d658b44b6814f8fe065aa/src/styles/styles.ts#L3985-L3987

Reply:
well, since it's philosophical question, I'll say you win.